static/src/stylesheets/module/facia/_item.scss

```diff
@@ -188,14 +188,19 @@ $fc-item-gutter: $gs-gutter / 4;
     }
 
     .u-faux-block-link--hover {
-        background-color: colour(neutral-6);
+        // For touch devices we want to reduce the time before the click event
+        // as much as possible, because browsers on touch devices give feedback
+        // at the end of the click event.
+        @include mq(desktop) {
```

Comment:
In my testing, with these hover styles and mouseover/mouseout, we incur a ~30ms repaint (due to background) and a 10ms layout (due to opacity, see https://twitter.com/OliverJAsh/status/636861689250582528). However, this all happens inside of an animation frame, which arrives after the click. Thus it is fine to keep these styles, because they do not increase the time between `touchend` and `click`.